srcs/builtins/echo_builtin.c

```diff
@@ -1,18 +1,55 @@
 /* ************************************************************************** */
 /*                                                                            */
 /*                                                        :::      ::::::::   */
-/*   echo.c                                             :+:      :+:    :+:   */
+/*   echo_builtin.c                                     :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
 /*   By: dmodrzej <dmodrzej@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2024/07/20 22:55:21 by dmodrzej          #+#    #+#             */
-/*   Updated: 2024/07/20 22:56:13 by dmodrzej         ###   ########.fr       */
+/*   Updated: 2024/07/25 00:36:11 by dmodrzej         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 
-int	builtin_echo(t_shell *shell, char **args)
+static void	identify_variable(t_shell *shell, char *str)
+{
+	char	*var_name;
+
+	while (*str && *str != ' ' && *str != '$')
```

Comment:
u can use ft_strchar for this.

Reply:
Yeah but the effect is still the same? Or not?

Reply:
Same =)